types/lhr.d.ts

```diff
@@ -61,7 +61,7 @@ declare global {
       /** The record of all formatted string locations in the LHR and their corresponding source values. */
       i18n: {rendererFormattedStrings: I18NRendererStrings, icuMessagePaths: I18NMessages};
       /** An array containing the result of all stack packs. */
-      stackPacks: Result.StackPack[];
+      stackPacks?: Result.StackPack[];
```

Comment:
wait this had no other impact being required the whole time? 😆

Reply:
> wait this had no other impact being required the whole time? 😆

Yeah, the only code using [was treating it as optional](https://github.com/GoogleChrome/lighthouse/blob/ca650db26339d05a63910bd240e269a683ee850e/lighthouse-core/report/html/renderer/util.js#L82) anyways, so easy change (or so I thought).